draft-ietf-tls-tls13.md

```diff
@@ -265,24 +264,23 @@ that has two basic properties:
   data encryption (e.g., AES {{AES}}, etc.).  The keys for
   this symmetric encryption are generated uniquely for each
   connection and are based on a secret negotiated by another
-  protocol (such as the TLS Handshake Protocol).  The Record
-  Protocol can also be used without encryption, i.e., in integrity-only
+  protocol (such as the TLS Handshake Protocol).  The TLS Record
+  Protocol can also be used without encryption to support integrity-only
```

Comment:
Null encryption is no longer permitted. (at least, after the first flights) I suggest just dropping this sentence.